addons/point_of_sale/static/src/js/Screens/TicketScreen/TicketScreen.js

```diff
@@ -83,10 +83,11 @@ odoo.define('point_of_sale.TicketScreen', function (require) {
             const screen = order.get_screen_data();
             if (['ProductScreen', 'PaymentScreen'].includes(screen.name) && order.get_orderlines().length > 0) {
                 const { confirmed } = await this.showPopup('ConfirmPopup', {
-                    title: 'Existing orderlines',
-                    body: `${order.name} has total amount of ${this.getTotal(
+                    title: this.env._t('Existing orderlines'),
+                    body: `${order.name} ${this.env._t(
+                        'has a total amount of')} ${this.getTotal(
```

Comment:
maybe you can use `_.str.sprintf` and `%s` to avoid splitting the sentence in small bits that are more difficult to translate

Reply:
I didn't know about it, thanks !